MANIFEST.in

```diff
@@ -1,2 +1,5 @@
 include ipinfo/countries.json
 include ipinfo/eu.json
+include ipinfo/flags.json
+include ipinfo/currency.json
+include ipinfo/continent.json
```

Comment:
Oh just FYI, we gotta ensure this also exists for nodejs and other places that have such a manifest.